cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -160,6 +164,7 @@ protected CruiseControl(HasMetadata resource) {
         this.mountPath = "/var/lib/kafka";
         this.logAndMetricsConfigVolumeName = LOG_AND_METRICS_CONFIG_VOLUME_NAME;
         this.logAndMetricsConfigMountPath = LOG_AND_METRICS_CONFIG_VOLUME_MOUNT;
+        this.isMetricsEnabled = DEFAULT_CRUISE_CONTROL_METRICS_ENABLED;
```

Comment:
I'm not sure if this boolean needs to exist if we are always equating it to 
```
 if (metrics != null)
```

Could always just gate the logic on that? (i.e. line 600)

Reply:
I agree but this field comes from history, I would avoid to change it now because removing it means refactoring other classes still using the field and they are not related to this PR. We should do a refactoring for that in a different PR.